src/Pos/Client/Txp/Balances.hs

```diff
@@ -65,39 +67,47 @@ type BalancesRedirect =
 runBalancesRedirect :: BalancesRedirect m a -> m a
 runBalancesRedirect = coerce
 
+type BalancesMonad ext m =
```

Comment:
Previously we had two `MonadBalances`, now we have two `MonadBalances` and one `BalancesMonad` :/